nifi-nar-bundles/nifi-standard-services/nifi-hbase-client-service-api/src/main/java/org/apache/nifi/hbase/HBaseClientService.java

```diff
@@ -67,6 +67,14 @@
             .defaultValue("1")
             .build();
 
+    PropertyDescriptor PHOENIX_CLIENT_JAR_LOCATION = new PropertyDescriptor.Builder()
+            .name("Phoenix Client JAR Location")
+            .description("The full path to the Phoenix client JAR. Required if Phoenix is installed on top of HBase.")
+            .addValidator(StandardValidators.FILE_EXISTS_VALIDATOR)
+            .expressionLanguageSupported(true)
+            .dynamicallyModifiesClasspath(true)
```

Comment:
Well, that is what I am trying to figure out. But regardless, _breaking_ simply means that without doing something extra it will no longer work. If this is the case then it's breaking, correct. I do understand that certain cases may be allowed for greater good and of course we would need to document them, but I just wanted to be clear on the implications.